src/main/java/io/pravega/connectors/flink/table/catalog/pravega/factories/PravegaCatalogFactoryOptions.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.connectors.flink.table.catalog.pravega.factories;
+
+import io.pravega.connectors.flink.dynamic.table.PravegaOptions;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.flink.formats.json.JsonOptions;
+import org.apache.flink.table.catalog.CommonCatalogOptions;
+
+/** {@link ConfigOption}s for {@link PravegaCatalogFactory}. */
+public class PravegaCatalogFactoryOptions {
+
+    public static final String IDENTIFIER = "pravega";
+
+    public static final ConfigOption<String> DEFAULT_DATABASE =
+            ConfigOptions.key(CommonCatalogOptions.DEFAULT_DATABASE_KEY).stringType().noDefaultValue();
+
+    public static final ConfigOption<String> CONTROLLER_URI = PravegaOptions.CONTROLLER_URI;
+
+    public static final ConfigOption<String> SCHEMA_REGISTRY_URI =
```

Comment:
Please write some description to further illustrate what is the configuration is for.